docs/reference/style/pack.rst

```diff
@@ -238,230 +238,90 @@ The weight of the font to be used.
+         <head>
+            <meta charset="UTF-8" />
+            <title>Pack layout testbed</title>
+            <style>
+               html, body {
+                  height: 100%;
+               }
+               body {
+                  overflow: hidden;
+                  display: flex;
+                  margin: 0;
+                  white-space: pre;
+               }
+               div {
+                  display: flex;
+                  white-space: pre;
+               }
+            </style>
+         </head>
+         <body></body>
+      </html>
+
+* The root element of the Pack layout can be mapped to the ``<body>`` element of
+  the HTML reference document. The rendering area of the browser window becomes
+  the view area that Pack will fill.
+
+* All other elements in the DOM tree are mapped to ``<div>`` elements.
+
+* The following Pack declarations can be mapped to equivalent CSS declarations:
+
+   ============================= ===================================================
+   Pack property                 CSS property
+   ============================= ===================================================
+   ``alignment: top``            ``align-items: start`` if ``direction == row``;
+                                 otherwise ignored.
+   ``alignment: bottom``         ``align-items: end`` if ``direction == row``;
+                                 otherwise ignored.
+   ``alignment: left``           ``align-items: start`` if ``direction == column``;
+                                 otherwise ignored.
+   ``alignment: right``          ``align-items: end`` if ``direction == column``;
+                                 otherwise ignored.
+   ``alignment: center``         ``align-items: center``
+   ``direction: <str>``          ``flex-direction: <str>``
+   ``display: pack``             ``display: flex``
+   ``flex: <int>``               If ``direction = row`` and ``width`` is set,
+                                 or ``direction = column`` and ``height`` is set,
+                                 ignore. Otherwise, ``flex: <int> 0 0``.
+   ``font_size: <int>``          ``font-size: <int>pt``
+   ``height: <int>``             ``height: <int>px``
+   ``padding_top: <int>``        ``margin-top: <int>px``
+   ``padding_bottom: <int>``     ``margin-bottom: <int>px``
+   ``padding_left: <int>``       ``margin-left: <int>px``
+   ``padding_right: <int>``      ``margin-right: <int>px``
+   ``text_direction: <str>``     ``direction: <str>``
+   ``width: <int>``              ``width: <int>px``
+   ============================= ===================================================
+
+* If ``direction = row``, and neither ``width`` nor ``flex`` is set, the element
+  assumes a CSS definition of ``flex: 0 0 0``.
+
+* If ``direction = column``, and neither ``height`` nor ``flex`` is set, the
+  element assumes a CSS definition of ``flex: 0 0 0``.
+
+* All other Pack declarations should be used as-is as CSS declarations, with
```

Comment:
```suggestion
* All other Pack declarations should be used as-is as CSS declarations, with
```
Since the initial value of `flex` is 0, this behavior is already covered by the `flex` definition above, as the `__css__` implementation shows. So these lines are redundant. This is consistent with the way we defined other properties with initial values, such as `alignment`.